app/services/profile_fields/add_base_fields.rb

```diff
@@ -3,11 +3,29 @@ class AddBaseFields
     include FieldDefinition
 
     group "Basic" do
```

Comment:
Added show_in_onboarding to name, website URL, summary and location . This will mean that new Forems ([post PR 10707](https://github.com/forem/forem/pull/10707/files)) will have these default fields to fill out in the profile step of onboarding modal. Note: Name will be removed in the [PR 10707](https://github.com/forem/forem/pull/10707/files). Please feel free to feedback if you don't think this makes sense.

Reply:
It may make sense to update `ProfileFields::ImportFromCsv` and `lib/data/dev_profile_fields.csv` accordingly too.

Reply:
Thanks @citizen428, I'd missed this file.
I've added the changes now. ✅